modules/tensor_mechanics/include/userobjects/HEVPInternalVarRateUOBase.h

```diff
@@ -0,0 +1,42 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef HEVPINTERNALVARRATEUOBASE_H
+#define HEVPINTERNALVARRATEUOBASE_H
+
+/**
+ * This user object is a pure virtual base classs
```

Comment:
If you move the comment immediately in front of the class declaration it will get picked up by doxygen